tests/kafkatest/tests/connect/connect_rest_test.py

```diff
@@ -34,9 +35,7 @@ class ConnectRestApiTest(KafkaTest):
 
     FILE_SOURCE_CONFIGS = {'name', 'connector.class', 'tasks.max', 'key.converter', 'value.converter', 'header.converter', 'batch.size',
                            'topic', 'file', 'transforms', 'config.action.reload', 'errors.retry.timeout', 'errors.retry.delay.max.ms',
-                           'errors.tolerance', 'errors.log.enable', 'errors.log.include.messages', 'predicates', 'topic.creation.groups',
-                           'offsets.storage.topic', 'transaction.boundary', 'transaction.boundary.interval.ms', 'config.action.reload',
-                           'exactly.once.support'}
+                           'errors.tolerance', 'errors.log.enable', 'errors.log.include.messages', 'predicates', 'topic.creation.groups'}
```

Comment:
Reverted to original since it is taken care in a separate PR - https://github.com/apache/kafka/pull/12575